projects/js-packages/ai-client/src/logo-generator/components/generator-modal.tsx

```diff
@@ -227,6 +235,7 @@ export const GeneratorModal: React.FC< GeneratorModalProps > = ( {
 		body = (
 			<>
 				{ ! logoAccepted && <Prompt initialPrompt={ initialPrompt } /> }
+				{ requireUpgrade && <FairUsageNotice /> }
```

Comment:
I think we need to check if the site is unlimited too. `requireUpgrade` can be true when tiers are enabled and the site is on the limit of the tier. In this case, showing the fair usage notice does not make sense.

Reply:
Yep, confirmed here.

<img width="500" alt="Screenshot 2024-08-30 at 12 24 26" src="https://github.com/user-attachments/assets/3ce4989e-fcea-4d03-b7e0-4f5d971c62d9">